CONTRIBUTING.md

```diff
@@ -376,15 +376,8 @@ will cause a broken ToC, and should be corrected to:
 
 ### Contributor T-shirt
 
-If your Pull Request to
-[Kong/docs.konghq.com](https://github.com/Kong/docs.konghq.com) was accepted,
-congratulations, you are eligible to receive the very special Contributor
-T-shirt! Go ahead and fill-out the [Contributors Submissions
-form](https://goo.gl/forms/5w6mxLaE4tz2YM0L2).
-
-Proudly wear your T-shirt and show it to us by tagging
-[@thekonginc](https://twitter.com/thekonginc) on Twitter!
-
-![Kong Contributor T-shirt](https://konghq.com/wp-content/uploads/2018/04/100-contributor-t-shirt-1024x768.jpg)
+Contributions to this repo that fix a bugs, adds functionality, or make it
+significantly easier to understand or use Kong will earn you a Kong contributor
+t-shirt! Find out how to request your shirt [here](https://github.com/Kong/kong/blob/master/CONTRIBUTING.md#contributor-t-shirt).
```

Comment:
t-shirt: T-shirt

Reply:
I believe that the previous phrasing read better. Also, we got rid of the "mention us on Twitter" and the picture, but should probably keep them.

> If your Pull Request to Kong/docs.konghq.com was accepted and fixes a bug, adds functionality, or makes it significantly easier to use or understand Kong, congratulations! You are eligible to receive the very special Contributor T-shirt! Go ahead and fill-out the Contributors Submissions form.

Reply:
The idea here was to reduce the number of places to update if something about the shirt changed, (for example the design or what to say in the tweet). 

I'm ok with keeping a single source of truth in the Kong repo or duplicating the content, but if there are other repos that will eventually qualify for shirts (ingress controller, pod injector for service mesh, etc.) then we might end up duplicating things a lot of places, and will have more to maintain when the shirt design gets a refresh.

Reply:
I was wondering if that was the intent. In this case, we can replace "contributions to Kong/docs.konghq.com" with "contributions to this repo", and end up with no more than a single paragraph we can copy paste in each repository, but more readable than the previous version of this paragraph. How does that sound?

Reply:
Maybe? I think this is more a theoretical question about what is part of Kong. If contributing to a Kong supported plugin gets a shirt, then maintaining your own open source plugin should also get you a shirt too. Maybe any integration should get you one (terraform / puppet modules, other stuff...) Not sure where we want to draw the line.

Reply:
>  If contributing to a Kong supported plugin gets a shirt, then maintaining your own open source plugin should also get you a shirt too.

I don't think so. The obvious line to draw is things included in the kong rockspec an external plugins: https://github.com/Kong/kong/blob/8ed77f7308d823b321ef713cea0868040f9257b9/kong-0.14.1-0.rockspec#L37
Though you *could* extend it if you wanted it to.

Reply:
I don't think its really up to me, it would probably be a Marco and Thibault discussion. Let's take it offline for now?

Reply:
For anyone following along at home, we will be including plugins when they are moved to a different repo.